src/components/carousel/carousel.js

```diff
@@ -402,7 +402,7 @@ export default {
     },
     touchMove(evt) /* istanbul ignore next: JSDOM doesn't support touch events */ {
       // ensure swiping with one touch and not pinching
-      if (evt.touches && evt.originalEvent.touches.length > 1) {
+      if (evt.touches && (evt.originalEvent || evt).touches.length > 1) {
```

Comment:
We can just use `if (evt.touches && evt.touches.length > 1) {`

Reply:
Yeah, this is a carryover from the Boostrap Without jQuery code (as they also have customized events in the non-jQuery version).

Reply:
@tmorehouse Cool, that's kinda what I figured. Thanks!